Sources/TensorFlow/Layers/Pooling.swift

```diff
@@ -424,3 +424,51 @@ public struct GlobalMaxPool3D<Scalar: TensorFlowFloatingPoint>: ParameterlessLay
     return input.max(squeezingAxes: [1, 2, 3])
   }
 }
+
+/// A fractional max pooling layer.
+///
+/// Note: `FractionalMaxPool` does not have an XLA implementation, and thus may have performance implications.
+@frozen
+public struct FractionalMaxPool<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  /// Pooling ratios for each dimension of input of shape (batch, height, width, channels).
+  /// Currently pooling in only height and width is supported.
+  @noDerivative public let poolingRatio: (Double, Double, Double, Double)
+  /// Determines whether pooling sequence is generated by pseudorandom fashion.
+  @noDerivative public let pseudoRandom: Bool
+  /// Determines whether values at the boundary of adjacent pooling cells are used by both cells
+  @noDerivative public let overlapping: Bool
+  /// Seed for the random number generator
+  @noDerivative public let seed: Int64
+  /// A second seed to avoid seed collision
+  @noDerivative public let seed2: Int64
+  /// Initializes a `FractionalMaxPool` layer with configurable `poolingRatio`.
+  public init(poolingRatio: (Double, Double, Double, Double), pseudoRandom: Bool = false,
```

Comment:
`poolingRatio.0` and `poolingRatio.3` always must be 1.0.
It would be good to have simplified initializer like `MaxPool2D` (at least from the point of view of consistency).
https://github.com/tensorflow/swift-apis/blob/010a390dc3a96836f73f801a1704dd2a9114e8a6/Sources/TensorFlow/Layers/Pooling.swift#L88-L101

Reply:
Thanks this was a a good idea!